resources/local/chainspec.toml

```diff
@@ -29,6 +29,14 @@ auction_delay = 3
 # The delay for the payout of funds, in eras. If a withdraw request is included in a block in era N
 # (other than the last one), they are paid out in the last block of era N + locked_funds_period.
 locked_funds_period = 15
+# Round seigniorage rate represented as a fractional number
```

Comment:
Sorry, I actually meant the first line of the comment:
```suggestion
# Round seigniorage rate represented as a fraction of the total supply
```

(It's a bit redundant to say something similar in the last line below; also in the other files.)